src/BenchmarkDotNet/Loggers/LinqPadLogger.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace BenchmarkDotNet.Loggers
+{
+    public sealed class LinqPadLogger : ILogger
+    {
+        private const string DefaultColor = "";
+
+        // Use Reflection to avoid taking a dependency on the LINQPad assembly
+        static readonly Type linqPadUtil = Type.GetType("LINQPad.Util, LINQPad, Version=1.0.0.0, Culture=neutral, PublicKeyToken=21353812cd2a2db5", throwOnError: true);
+        private static readonly MethodInfo withStyle = linqPadUtil.GetMethod("WithStyle", BindingFlags.Static | BindingFlags.Public);
+
+        public static bool IsAvailable => AppDomain.CurrentDomain.FriendlyName.StartsWith("LINQPad Query Server", StringComparison.OrdinalIgnoreCase);
+
+        public static ILogger Default { get; } = new LinqPadLogger();
+
+        private readonly IReadOnlyDictionary<LogKind, string> colorScheme;
+
+        public LinqPadLogger()
+        {
+            var isDarkTheme = (bool) linqPadUtil.GetProperty("IsDarkThemeEnabled", BindingFlags.Static | BindingFlags.Public).GetValue(null);

```

Comment:
Did you check how it works in old LINQPad versions which don't have this property? This case should be handled correctly without any additional exceptions.

Reply:
Will fix this to handle `GetProperty` returning `null` (i.e., property not found).